src/lib/openapi/spec/project-overview-schema.ts

```diff
@@ -17,42 +17,60 @@ export const projectOverviewSchema = {
     properties: {
         stats: {
             $ref: '#/components/schemas/projectStatsSchema',
+            description: 'Project statistics',
         },
         version: {
             type: 'number',
+            example: 1,
+            description: '',
```

Comment:
I'm assuming this is what you're looking for more info on, right?

Reply:
yes, I leave the descriptions empty in case I don't know what the property is or if some property doesn't show up in the runtime data, but is part of the schema

Reply:
Just so I'm clear: are you intending to merge this with an empty description?

Reply:
no, removing this one for now